### PR TITLE
Bumped up Pulse's Observatory score

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #875 

Not sure what to do with the SRI part. I couldn't generate integrity for [`https://platform.twitter.com/widgets.js`](https://github.com/mozilla/network-pulse/blob/master/server.js#L94) using [SRI Hash Generator](https://www.srihash.org/)

> Error: this resource is not eligible for integrity checks. See http://enable-cors.org/server.html

### CODE REVIEW

**Comment by mmmavis** [comment]
@cadecairos got time to review this PR this week? 👀 😁

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "Pomax",
+      "mavis"
+    ]
+  },
+  "bugs": {
+    "list": "https://github.com/mozilla/network-pulse/issues",
+    "report": "https://github.com/mozilla/network-pulse/issues/new"
+  },
+  "urls": {
+    "prod": "network-pulse-staging.herokuapp.com",
```
Comment (cadecairos) [inline contribute.json]: this should be `https://www.mozillapulse.org/`

File: server.js [hunk 2]
```diff
@@ -32,9 +32,10 @@ app.use(helmet.xssFilter({
   setOnOldIE: true
 }));
 
-// maxAge for HSTS header must be at least 18 weeks (see https://hstspreload.org/)
+// maxAge for HSTS header must be at least 6 months
+// (see https://wiki.mozilla.org/Security/Guidelines/Web_Security#HTTP_Strict_Transport_Security)
 app.use(helmet.hsts({
-  maxAge: 60 * 60 * 24 * 7 * 18, // 18 weeks in seconds
+  maxAge: 15768000, // 6 months in seconds
```
Comment (cadecairos) [inline server.js]: lets make this a year

**Comment by mmmavis** [comment]
@cadecairos here we go: https://network-pulse-staging-pr-876.herokuapp.com/contribute.json

**Comment by mmmavis** [comment]
Yay! Merging! 🍻

**Pull request merged**